PyDSS/modes/QSTS.py

```diff
@@ -15,11 +15,11 @@ def __init__(self, dssInstance, settings: SimulationSettingsModel, Logger):
 
         start_time_hours = self._Hour + self._Second / 3600.0
         load_shape_resolutions_secs = get_load_shape_resolution_secs()
-        if load_shape_resolutions_secs == self._sStepRes:
-            # I don't know why this is needed in this case.
-            # The first data point gets skipped without it.
-            # FIXME
-            start_time_hours += self._sStepRes / 3600.0
+        #if load_shape_resolutions_secs == self._sStepRes:
+        #    # I don't know why this is needed in this case.
+        #    # The first data point gets skipped without it.
+        #    # FIXME
+        #    #start_time_hours += self._sStepRes / 3600.0
```

Comment:
@AadilLatif @nadiavp We will need to discuss how to solve this. I will schedule a meeting.